Source/SelfService/Web/layout/layoutWithSidebar.tsx

```diff
@@ -101,28 +62,81 @@ export const LayoutWithSidebar: React.FunctionComponent<Props> = (props) => {
 
 export const getDefaultMenuWithItems = (
     history: History<LocationState>,
-    items: any[]
+    topItems: any[],
+    bottomItems: any[]
```

Comment:
Would you consider renaming `topItems` => `navigationItems`, and `bottomItems` => `actionItems`, or something similar. 

The reason is I usually try to lean on the specific semantics of the properties as much as possible, and not focus on the pure visual aspect, like top/bottom. Now, this isn't something that's always avoidable, and for purely visual components then it makes sense to use these visual terms.

Thoughts?

Reply:
Done!